### DISCUSSION TITLE
DataTable - move focus programatically

### DISCUSSION THREAD

**Discussion by kanalje** [primefaces/community#865] [PrimeVue]
Is there a way to move focus to cell given a row index and column name/index?

```
<Column field="quantity" :header="t('Quantity')" :sortable="false" style="width: 10%" class="quantity">
  <template #editor="{data, field}">
    <q-input dense outlined v-model="data[field]" class="q-my-none q-py-none" />
  </template>
</Column>
```

The editor template isn't rendered until the cell is focused so I can't use the input field.

I'm using DataTable to display and edit order lines in an order.
After adding a new orderline I want the second column of that row to be focused.

Edit:
If I fire a click event on the <td> element I can see the cell-edit-init event being fired and the q-input component briefly being slotted in, but then immediately after, cell-edit-complete is fired.

Edit2: Never mind. A small timeout on the the table cell click did the trick.
```
setTimeout(() => {
  tableCell.click();
}, 100);
```

**Comment by fwt-parth-jariwala** [comment]
I am also implementing similar functionality like you. I have attached cell-edit-completed and checked if that cell is last cell and that row is last row, then adding new empty row below it. Now i want to focus on second column of newly added row..

Have you found any solution for it??
I am currently using vue 3.4.21 and primevue 3.52.0
